build.xml

```diff
@@ -1,38 +1,148 @@
 <?xml version="1.0"?>
```

Comment:
This built okay last time I used it a few months ago, I don't think anything has changed since then. Testing it on E1 now.